pkg/issuer/acme/dns/dns.go

```diff
@@ -155,6 +156,23 @@ func (s *Solver) solverFor(crt *v1alpha1.Certificate, domain string) (solver, er
 		if err != nil {
 			return nil, fmt.Errorf("error instantiating route53 challenge solver: %s", err.Error())
 		}
+	case providerConfig.RFC2136 != nil:
+		tsigSecret, err := s.secretLister.Secrets(s.resourceNamespace).Get(providerConfig.RFC2136.TSIGSecret.Name)
+		if err != nil {
+			return nil, fmt.Errorf("error getting rfc2136 service account: %s", err.Error())
+		}
```

Comment:
It looks like if this is left blank, authentication can be disabled. If the secret specified doesn't exist we should attempt without auth.